src/test/scala/ru/yandex/qatools/allure/scalatest/AllureReporterSpec.scala

```diff
@@ -33,9 +35,9 @@ class AllureReporterSpec extends FlatSpec with BeforeAndAfter {
     allure = mock(classOf[Allure])
     reporter = spy(new AllureReporter)
     reporter.setLifecycle(allure)
-    doReturn(testUuid).when(reporter).getSuiteUuid(any(classOf[String]))
```

Comment:
With mockito-core 2.4.2 the old form ends up an ambiguous overload at compile time.